.travis.yml

```diff
@@ -14,13 +14,18 @@ env:
 matrix:
   allow_failures:
   - go: master
+  exclude:
+  - python: 3.6
 
 before_install:
```

Comment:
We should probably throw a comment in here or somewhere that this is a specific step for travis, or maybe we rename this `travis_setup`.